src/main.rs

```diff
@@ -15,12 +15,33 @@ struct Opt {
 
     /// UFO source file paths
     #[structopt(help = "UFO source path(s)")]
-    ufopaths: Vec<String>,
+    ufopaths: Vec<PathBuf>,
 }
 
 fn main() {
     let argv = Opt::from_args();
 
+    // ~~~~~~~~~~~~~~~~~~~~~~
+    // UFO dir validity check
+    // ~~~~~~~~~~~~~~~~~~~~~~
+    let mut bad_ufo_path = false;
+    for ufo_testpath in &argv.ufopaths {
+        match ufo_testpath.exists() {
+            true => (),
+            false => {
+                let error_str = "[ERROR]".red().bold();
+                eprintln!("{} {:?} is not a valid UFO directory path", error_str, ufo_testpath);
+                bad_ufo_path = true;
```

Comment:
You could also exit right here.

Reply:
Trying to capture all invalid paths if there are > 1 UFO source file args.  Know of a better approach?

Reply:
e.g.,  with five bad ufo paths

```
target/release/ufofmt 1 2 3 4 5
[ERROR] "1" is not a valid UFO directory path
[ERROR] "2" is not a valid UFO directory path
[ERROR] "3" is not a valid UFO directory path
[ERROR] "4" is not a valid UFO directory path
[ERROR] "5" is not a valid UFO directory path
```

Reply:
Ah, right. One could probably do a https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter and collect all the bad paths for display and immediate exit.

Reply:
> Ah, right. One could probably do a https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter and collect all the bad paths for display and immediate exit.

I like it!  I'll give it a try

Reply:
You should have full maintainer access btw.  Please feel free to add any changes that you'd like to see in a formatter client frontend for the lib.

Reply:
Thanks! I may be silent for a while because there is so much stuff to do elsewhere, though.

Reply:
No worries at all!  Don't feel compelled to do anything.  Just making sure that you know that you are always welcome to contribute here.

Reply:
> Ah, right. One could probably do a https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter and collect all the bad paths for display and immediate exit.

466707a7bbe4e10b109c3885aa8c45b9c581615f